packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/DiagramEventProcessor.java

```diff
@@ -164,6 +168,13 @@ public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDesc
                         .layoutData(layoutData)
                         .build();
 
+                for (var diagramPostProcessor: this.diagramPostProcessors) {
+                    this.diagramContext = new DiagramContext(laidOutDiagram);
+                    if (diagramPostProcessor.canHandle(this.editingContext, this.diagramContext)) {
+                        laidOutDiagram = diagramPostProcessor.postProcess(this.editingContext, this.diagramContext).orElse(laidOutDiagram);
+                    }
+                }
```

Comment:
We are trying very hard to remove stuff from the event processors, not to add any. I suspect that this could be removed from here entirely and it would create a PR way smaller. I'll look into it.